fe/pages/elastic_table.py

```diff
@@ -19,7 +19,19 @@
         "sortable",
         "default_sort",
     ],
-    defaults=[None, None, lambda x: x, None, False, False, False],
+    defaults=[
+        None,
+        None,        
+        lambda x: (
+            f"{x:.2e}" if isinstance(x, float) and not x.is_integer() and abs(x) < 0.01 # Use scientific notation for small floats
+            else f"{x:.3f}" if isinstance(x, float) and not x.is_integer() # Use fixed-point notation for larger floats
+            else x # Keep other types as is
+        ),
```

Comment:
Actually in this case, for readability, I think it would be better if it was a separate top level function, something like `def format_float(x)`; which could then be passed here in the list of defaults